library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -41,6 +41,7 @@ $secondary-color:  			 #F00 !default;
 $alt-colour: 				 #d36f00 !default;
 
 $text-color:				 #212121 !default;
+$text-color-disabled:		 lighten($text-color, 30%) !default;
```

Comment:
I would prefer this to be written as hex value, since we are trying to get it as light as possible and still be accessible. #767676 seems like the best value.